diff -Nur -x Debug -x Release -x '*.user' -x .deps -x build --binary ./libxml2-2.9.3/include/libxml/xmlversion.h ./libxml2-2.9.3.vc14/include/libxml/xmlversion.h
--- ./libxml2-2.9.3/include/libxml/xmlversion.h	2015-11-20 12:49:39.000000000 +0300
+++ ./libxml2-2.9.3.vc14/include/libxml/xmlversion.h	2016-04-25 18:04:38.372998900 +0300
@@ -174,7 +174,7 @@
  *
  * Whether the FTP support is configured in
  */
-#if 1
+#if 0
 #define LIBXML_FTP_ENABLED
 #endif
 
@@ -183,7 +183,7 @@
  *
  * Whether the HTTP support is configured in
  */
-#if 1
+#if 0
 #define LIBXML_HTTP_ENABLED
 #endif
 
@@ -396,7 +396,7 @@
  *
  * Whether the Zlib support is compiled in
  */
-#if 1
+#if 0
 #define LIBXML_ZLIB_ENABLED
 #endif
 
@@ -405,7 +405,7 @@
  *
  * Whether the Lzma support is compiled in
  */
-#if 1
+#if 0
 #define LIBXML_LZMA_ENABLED
 #endif
 
diff -Nur -x Debug -x Release -x '*.user' -x .deps -x build --binary ./libxml2-2.9.3/win32/VC10/config.h ./libxml2-2.9.3.vc14/win32/VC10/config.h
--- ./libxml2-2.9.3/win32/VC10/config.h	2015-05-06 15:04:36.000000000 +0300
+++ ./libxml2-2.9.3.vc14/win32/VC10/config.h	2016-04-25 17:52:42.339027800 +0300
@@ -18,6 +18,10 @@
 #define HAVE_STDLIB_H
 #define HAVE_TIME_H
 #define HAVE_FCNTL_H
+
+#define SEND_ARG2_CAST
+#define GETHOSTBYNAME_ARG_CAST
+
 #include <io.h>
 #include <direct.h>
 #endif
@@ -96,8 +100,8 @@
 
 #if defined(_MSC_VER)
 #define mkdir(p,m) _mkdir(p)
-#define snprintf _snprintf
 #if _MSC_VER < 1500
+#define snprintf _snprintf
 #define vsnprintf(b,c,f,a) _vsnprintf(b,c,f,a)
 #endif
 #elif defined(__MINGW32__)
diff -Nur -x Debug -x Release -x '*.user' -x .deps -x build --binary ./libxml2-2.9.3/win32/VC10/libxml2.sln ./libxml2-2.9.3.vc14/win32/VC10/libxml2.sln
--- ./libxml2-2.9.3/win32/VC10/libxml2.sln	2015-05-06 15:04:36.000000000 +0300
+++ ./libxml2-2.9.3.vc14/win32/VC10/libxml2.sln	2016-04-25 16:29:35.822890300 +0300
@@ -8,7 +8,7 @@
 		{BEC03130-A64C-48F8-863C-52DA83CFB3AE} = {BEC03130-A64C-48F8-863C-52DA83CFB3AE}
 	EndProjectSection
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "iconv", "..\..\..\libiconv-1.14.vc10\windows\VC10\iconv.vcxproj", "{BEC03130-A64C-48F8-863C-52DA83CFB3AE}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "iconv", "..\..\..\libiconv-1.14\windows\VC10\iconv.vcxproj", "{BEC03130-A64C-48F8-863C-52DA83CFB3AE}"
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
diff -Nur -x Debug -x Release -x '*.user' -x .deps -x build --binary ./libxml2-2.9.3/win32/VC10/libxml2.vcxproj ./libxml2-2.9.3.vc14/win32/VC10/libxml2.vcxproj
--- ./libxml2-2.9.3/win32/VC10/libxml2.vcxproj	2015-05-06 15:04:36.000000000 +0300
+++ ./libxml2-2.9.3.vc14/win32/VC10/libxml2.vcxproj	2016-06-01 16:24:16.589719100 +0300
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Label="ProjectConfigurations">
     <ProjectConfiguration Include="Debug|Win32">
       <Configuration>Debug</Configuration>
@@ -13,18 +13,21 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{1D6039F6-5078-416F-A3AF-A36EFC7E6A1C}</ProjectGuid>
     <RootNamespace>libxml2</RootNamespace>
+    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>StaticLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
     <CharacterSet>Unicode</CharacterSet>
+    <PlatformToolset>v140</PlatformToolset>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
-    <ConfigurationType>Application</ConfigurationType>
+    <ConfigurationType>StaticLibrary</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
     <WholeProgramOptimization>true</WholeProgramOptimization>
-    <CharacterSet>MultiByte</CharacterSet>
+    <CharacterSet>Unicode</CharacterSet>
+    <PlatformToolset>v140</PlatformToolset>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
   <ImportGroup Label="ExtensionSettings">
@@ -46,10 +49,11 @@
     <ClCompile>
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
-      <AdditionalIncludeDirectories>$(ProjectDir);$(ProjectDir)..\..\include;$(ProjectDir)..\..\..\libiconv-1.14.vc10\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(ProjectDir);$(ProjectDir)..\..\include;$(ProjectDir)..\..\..\libiconv-1.14\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <DisableSpecificWarnings>4996</DisableSpecificWarnings>
       <MinimalRebuild>false</MinimalRebuild>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
+      <PreprocessorDefinitions>_UNICODE;UNICODE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
       <GenerateDebugInformation>true</GenerateDebugInformation>
@@ -61,6 +65,9 @@
       <Optimization>MaxSpeed</Optimization>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
+      <PreprocessorDefinitions>_UNICODE;UNICODE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>$(ProjectDir);$(ProjectDir)..\..\include;$(ProjectDir)..\..\..\libiconv-1.14\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <DisableSpecificWarnings>4996</DisableSpecificWarnings>
     </ClCompile>
     <Link>
       <GenerateDebugInformation>true</GenerateDebugInformation>
@@ -122,7 +129,7 @@
     <ClInclude Include="config.h" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\libiconv-1.14.vc10\windows\VC10\iconv.vcxproj">
+    <ProjectReference Include="..\..\..\libiconv-1.14\windows\VC10\iconv.vcxproj">
       <Project>{bec03130-a64c-48f8-863c-52da83cfb3ae}</Project>
       <Private>true</Private>
       <ReferenceOutputAssembly>true</ReferenceOutputAssembly>
diff -Nur -x Debug -x Release -x '*.user' -x .deps -x build --binary ./libxml2-2.9.3/win32/VC10/libxml2.vcxproj.filters ./libxml2-2.9.3.vc14/win32/VC10/libxml2.vcxproj.filters
--- ./libxml2-2.9.3/win32/VC10/libxml2.vcxproj.filters	2015-05-06 15:04:36.000000000 +0300
+++ ./libxml2-2.9.3.vc14/win32/VC10/libxml2.vcxproj.filters	2016-04-25 16:07:33.941468900 +0300
@@ -153,6 +153,9 @@
     <ClCompile Include="..\..\HTMLtree.c">
       <Filter>Source Files</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\buf.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
   </ItemGroup>
   <ItemGroup>
     <ClInclude Include="config.h">
diff -Nur -x Debug -x Release -x '*.user' -x .deps -x build --binary ./libxml2-2.9.3/win32/VC10/runsuite.vcxproj ./libxml2-2.9.3.vc14/win32/VC10/runsuite.vcxproj
--- ./libxml2-2.9.3/win32/VC10/runsuite.vcxproj	2015-05-06 15:04:36.000000000 +0300
+++ ./libxml2-2.9.3.vc14/win32/VC10/runsuite.vcxproj	2016-06-01 16:01:00.616456700 +0300
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Label="ProjectConfigurations">
     <ProjectConfiguration Include="Debug|Win32">
       <Configuration>Debug</Configuration>
@@ -13,18 +13,21 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{D841B4F7-C277-49CB-B379-8B1801C8EE35}</ProjectGuid>
     <RootNamespace>runsuite</RootNamespace>
+    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
     <CharacterSet>Unicode</CharacterSet>
+    <PlatformToolset>v140</PlatformToolset>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
     <WholeProgramOptimization>true</WholeProgramOptimization>
-    <CharacterSet>MultiByte</CharacterSet>
+    <CharacterSet>Unicode</CharacterSet>
+    <PlatformToolset>v140</PlatformToolset>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
   <ImportGroup Label="ExtensionSettings">
@@ -44,7 +47,7 @@
     <ClCompile>
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
-      <AdditionalIncludeDirectories>$(ProjectDir);$(ProjectDir)..\..\include;$(ProjectDir)..\..\..\libiconv-1.14.vc10\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(ProjectDir);$(ProjectDir)..\..\include;$(ProjectDir)..\..\..\libiconv-1.14\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <DisableSpecificWarnings>4996</DisableSpecificWarnings>
       <PreprocessorDefinitions>_UNICODE;UNICODE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <AdditionalOptions>/MP %(AdditionalOptions)</AdditionalOptions>
@@ -72,7 +75,7 @@
     <ClCompile Include="..\..\runsuite.c" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\libiconv-1.14.vc10\windows\VC10\iconv.vcxproj">
+    <ProjectReference Include="..\..\..\libiconv-1.14\windows\VC10\iconv.vcxproj">
       <Project>{bec03130-a64c-48f8-863c-52da83cfb3ae}</Project>
     </ProjectReference>
     <ProjectReference Include="libxml2.vcxproj">
